src/spacegame3/SpaceGame.java

```diff
@@ -80,29 +94,45 @@ public void giveSceneTo(SizableScene scene){
         stage.setScene(scene);
     }
 
+    /*
+        *main method - runs + launches game
+    */
     public static void main(String[] args) {
         launch(args);
     }
 
+    //getter for game started
```

Comment:
use /** */ for methods (just repeating for each one), also don't need to specify that it is a getter